src/pages/smarttax/tax-information-by-state/[slug].astro

```diff
@@ -47,5 +49,13 @@ const { Content, headings } = await entry.render();
       <i>Updated {format_date(entry.data.updated)}</i>
     </p>
   </main>
+  <!-- New duplicate section only shown on mobile. -->
+  <SinglePageSideNav
+      title={entry.data.name}
+      headings={headings}
+      parent_nav_path = "smarttax/state-tax-forms"
```

Comment:
I think we want the new url here too: 
` parent_nav_path = "smarttax/tax-information-by-state"`

Reply:
Nice catch.  Updated the parent navigation path. I also updated the paths in the FAQ as well.